tamr_toolbox/models/attribute_type.py

```diff
@@ -0,0 +1,152 @@
+"""Enum and dataclasses representing attribute types in Tamr"""
+from enum import Enum
+from dataclasses import dataclass
+from typing import ClassVar, Tuple, Union
+from tamr_toolbox.models.data_type import JsonDict
+from copy import deepcopy
+
+PrimitiveType = Enum("PrimitiveType", ["BOOLEAN", "DOUBLE", "INT", "LONG", "STRING"])
+
+# primitive type aliases
+DOUBLE = PrimitiveType.DOUBLE
+BOOLEAN = PrimitiveType.BOOLEAN
+INT = PrimitiveType.INT
+LONG = PrimitiveType.LONG
+STRING = PrimitiveType.STRING
+
+
+@dataclass(frozen=True)
+class Array:
```

Comment:
should we add this to the docs as well.

Reply:
What did you have in mind? I could see a paragraph or two introducing Attribute Types in Tamr being useful and a link to our online documentation for more info?

Reply:
Oh i meant something very simple. In the models.rst file under docs_src. Couldn't figure out how to add it here. Otherwise it looks good to me. 
Attribute Type
------------------------
.. automodule:: tamr_toolbox.models.attribute_type